openedx_certificates/processors.py

```diff
@@ -0,0 +1,224 @@
+"""
+This module contains processors for certificate criteria.
+
+The functions prefixed with `retrieve_` are automatically detected by the admin page and are used to retrieve the
```

Comment:
> by the admin page

I would have thought it's a cron job calling these, not manual trigger from the admin page?